src/core.ts

```diff
@@ -128,10 +134,13 @@ export default class AgentCore extends Store {
     })
 
     if (this._baseUrl && (query.length >= MAX_INTERACT || force)) {
+      const customState: any = this.get('custom')
```

Comment:
`s/any/Object/` ?

Reply:
`customeState` will use to `obj2query`. the function need argument Object __value typing__ for accept string function, `encodeURIComponent`. 
So Object can not assign `obj2query` arg.